survey_form/index.html

```diff
@@ -43,8 +43,196 @@
+      />
+
+      <label class="form-survey__label" for="age">Age: </label>
+      <input
+        type="age"
+        id="age"
+        class="form-survey__input"
+        placeholder="44"
+        min="12"
+        max="123"
+        required
+      />
+
+      <label class="form-survey__label" for="current_role"
+        >Which option best describes your current role?</label
+      >
+
+      <select class="form-survey__dropdown" id="current_role">
+        <option value="select_an_option" disabled>Select an option</option>
+        <option value="student">Student</option>
+        <option value="full_time_job">Full time job</option>
+        <option value="full_time_learner">Full time learner</option>
+        <option value="prefer_not_to_say">Prefer not to say</option>
+        <option value="other">Other</option>
+      </select>
+
+      <legend class="form-survey__label form-survey__label--radiobutton">
+        How likely is that you would recommend freeCodeCamp to a friend?
+      </legend>
+
+      <div class="form-survey__radiobutton--section">
+        <div class="form-survey__radiobutton--wrapper">
+          <input
+            class="form-survey__input--radiobutton"
+            type="radio"
+            name="recommendation"
+            id="definitely"
+          />
+          <label class="form-survey__label--recommendation" for="definitely"
+            >Definitely</label
+          >
+        </div>
+
+        <div class="form-survey__radiobutton--wrapper">
+          <input
+            class="form-survey__input--radiobutton"
+            type="radio"
+            name="recommendation"
+            id="maybe"
+          />
+          <label class="form-survey__label--recommendation" for="maybe"
+            >Maybe</label
+          >
+        </div>
+
+        <div class="form-survey__radiobutton--wrapper">
+          <input
+            class="form-survey__input--radiobutton"
+            type="radio"
+            name="recommendation"
+            id="not_sure"
+          />
+          <label class="form-survey__label--recommendation" for="not_sure"
+            >Not sure</label
+          >
+        </div>
+      </div>
+
+      <label class="form-survey__label" for="preferences"
+        >What do you like most in FCC:</label
+      >
+
+      <select class="form-survey__dropdown" id="preferences">
+        <option value="select_an_option" disabled>Select an option</option>
+        <option value="challenges">Challenges</option>
+        <option value="projects">Projects</option>
+        <option value="community">Community</option>
+        <option value="open_source">Open source</option>
+      </select>
+
+      <label
+        class="form-survey__label form-survey__label--radiobutton"
+        for="plans"
+        >Things that should be improved in the future (Check all that
+        apply):</label
+      >
+
+      <div class="form-survey__checkbox" id="plans">
+        <div class="form-survey__checkbox--wrapper">
```

Comment:
jw. `form-survey__checkbox--wrapper` => `form-survey__checkbox-wrapper`, to samo w innych wystąpieniach.